pkg/taxonomy/schema_test.go

```diff
@@ -0,0 +1,23 @@
+package taxonomy
+
+import (
+	"testing"
+
+	. "github.com/onsi/gomega"
+)
+
+// Validate a sample validation schema successfully
+func TestValidateSchema_simple_success(t *testing.T) {
+	taxonomyFile := "../../test/taxonomy/catalog.structs.schema.json"
+	g := NewGomegaWithT(t)
+	err := ValidateSchema("./" + taxonomyFile)
+	g.Expect(err).NotTo(HaveOccurred())
+}
+
+// Validate a sample badly formed validation schema
+func TestValidateSchema_simple_fail(t *testing.T) {
+	taxonomyFile := "../../test/taxonomy/bad_catalog.structs.schema.json"
+	g := NewGomegaWithT(t)
+	err := ValidateSchema("./" + taxonomyFile)
```

Comment:
Why the `./` prefix to an already relative path? Same for the TestValidateSchema_simple_success function.

Reply:
Right.